.travis.yml

```diff
@@ -17,3 +17,11 @@ env:
   matrix:
     secure: r8OGdpfg+XYSakj6YvFyOm5ll7Dm1h1SW17fhUvmSAFin1nvlTWGiORgt0Jhe6OqKHN97WHRZz6ZWjVPbcHlKQD1Py5u4o7IENXf5BOAX7jKESFXv5Ecbg65b4oQJbUuHDtQzA1hbQwSzU012eByqJhEwQ0hnoJYGP9PUhCg+NaGHpjLiMCf8KpjrugUntFc2zox0CD5MFC60VvtflIe3ix3T5Vupt1evyQ11Nt25++Sd/D212WxVxGWOaSbEr8IQ1QEkwOKXV1k6Vq5dHbaJ+OujVPGSm2HiiJGc/KW7wxjTr/LfEg8WngQpocrws3+gYAWmL28IO+oggitBOFi9Fzy/1xEQHnD3mtbx3P7iFWqMqx6QOadRZyGOrlpkACURpnL0gY76tdhcgs8HtvQO2G/Ht7hhmDkaXXxuImFjiSIe6ruYRzpK691CE3Kw4BmYTq+RDZayYq82l/5uamZbwSzxfQ3J+ZQazSCl9nqFVqV7b8NNM9Xlo3CFy8r5KLZechSIwXQzi/c6YpsoJ5unbLX1T36pGeIXULmcPBNgHstt9IjdJGVjvpolG56bW8G3B12gVoTdCigtUgDvjAYRRF052EsBpU/YCAIA2UZockbfdRPprMqVUGgeOC0vH/OfIWFcVwqudofrzYXRJA3ykXBoYaVwwWEC1i02cn0EUY=
 after_success:
+
+notifications:
+  slack:
+    on_pull_requests: false
+    on_success: never
+    on_failure: always
+    rooms:
+     - secure: "YoSSxftifz8V8BIGXQSAJTg+mkKs+QVYrnSJ7U4r1M4oP8QUgzBhgv8oBoA35uUA9YqLUi5kYqF7ClRiGg/UAzECEb9svH3Y1ipCvTr/Sz7DEOA960ZgqiEduP8YZARPxJo6aGUR1t8ZF9t1mVK9lFiNchuvZ6+GqKhxpvCkojE3bFZj+0QIw1OuTFdApR7j+ZTazCB9iE6Wx92Jfad0KS47C9jr7+7IF03eSN7VOsLUNQlP659TTeUMbHZ0IcmhalazTtgDiSRuUjYfBTzhKB/gYAVFfccNNx7fQGUHUu29Qm6JKenyZq1BxNNw/CAMvwFePmBukdVaVTdn5YVLe1v3uERnzYd/vbGHdTW8t1Dh13zXtZA4mERRDpSOeyp1MSmJDBg7iKG4UmtpcpuTHQr/1n12BV2wjMLw3VYdQym8t/uYFsnaKcJf9CSR2Rz9B1laLDIxAgzcyjksupKSmMuT1FQiKDAGQCov5UBxvGpSz7MP4fgSe7/de2ypxnLZvrmgWsuriLXgKgXjtGhhr/BKhXkwZc7rmxjJ0ZJodmLIbePG7ylFH/e4B2rBwApMTr518/Zh6uQIXExxmUq9fNb5c9ObReU04DVded6uqy2bnIQMdjZ3oevuXCdwZiIjAyKpLx3WSsYs3pTfLgLkyGhNkwoqO/NTew7bt6T9l40="
```

Comment:
Just out of curiosity, any reason this is value is surrounded by double quotes?

All other `secure` values i've seen have omitted them.

Reply:
I followed the travis documentation from here - https://docs.travis-ci.com/user/notifications/#Configuring-Slack-notifications
In the examples, they have the double quotes for some reason.